src/app/timetable/components/week-days/week-days.component.html

```diff
@@ -1,29 +1,29 @@
 <div class="saptamana">
-    <div class="zile">
-        <button class="buton">MON</button>
-    </div>
+  <div class="zile">
+    <button class="buton" (click)="updateDay($event)">MON</button>
```

Comment:
You can pass as a parameter the name of the day and get rid of `switch case` and `$event`